src/engine/controls/bpmcontrol.cpp

```diff
@@ -381,6 +381,8 @@ double BpmControl::shortestPercentageChange(const double& current_percentage,
 }
 
 double BpmControl::calcSyncedRate(double userTweak) {
+    //qDebug() << getGroup() << "BpmControl::calcSyncedRate, tweak " << userTweak;
+    m_dUserRateTweak = userTweak;
```

Comment:
maybe this should also be atomic?  I don't think different threads call calcSyncedRate and updateBeatDistance

Reply:
Yes, this is only called from the engine thread calculateSpeed()